tst/testspecial/run_all.sh

```diff
@@ -17,5 +28,5 @@ for gfile in *.g; do
         rm -f "${gfile}.bad"
     fi;
 done;
-exit ${retvalue}
```

Comment:
Why remove this? If we really want to loose it, please also remove `retvalue`, which is now unused.